compose.yml

```diff
@@ -0,0 +1,105 @@
+# yredis demo stack - exposed services:
+#
+#   http://localhost:2222/    demos/auth-express app
+#   http://localhost:3333/    demos/blocksuite app
+#   http://localhost:3002/    yr-server (ydoc sync)
+#
+# these are just exposed for troubleshooting:
+#
+#   http://localhost:5432/    Postgres
+#   http://localhost:8001/    Redis analytics
+#   http://localhost:9001/    MinIO console
+
+x-vars:
+  shared: &shared
+    env_file: .env
+    # overrides to connect services inside of Docker overlay network
+    environment: &shared-env
```

Comment:
This is a good idea!

If the compose file overrides the configurations, then we only need to generate the auth tokens in `.env`